integration/terraform/ec2/linux/main.tf

```diff
@@ -21,6 +21,7 @@ resource "aws_instance" "integration-test" {
       "sudo cp original.pem /opt/aws/amazon-cloudwatch-agent/original.pem",
       "sudo cp combine.pem /opt/aws/amazon-cloudwatch-agent/combine.pem",
       "export LOCAL_STACK_HOST_NAME=${var.local_stack_host_name}",
+      "export AWS_REGION=${var.region}",
```

Comment:
Why is this needed. Will it not default to the region the instance is located?

Reply:
The SDK needs a region configuration: https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-the-aws-region

The AMI cleaner runs on the GitHub runner, which has the AWS_REGION value set from a previous action, so that works. The integration tests are run on the EC2 instance, so this isn't set at all.